process-automation/latest/model/projects.md

```diff
@@ -30,17 +30,20 @@ The properties for a project are:
 
 | Property | Description |
 | -------- | ----------- |
-| Project name | *Required.* A unique name for a project. Project names must be in lowercase and between 1 and 26 characters in length. Alphanumeric characters and hyphens are allowed, however the name must begin with a letter and end alphanumerically, for example: `project-4-a` |
-| Project description | *Optional.* A short description of what the project is for. |
+| Name | *Required.* A unique name for a project. Project names must be in lowercase and between 1 and 26 characters in length. Alphanumeric characters and hyphens are allowed, however the name must begin with a letter and end alphanumerically, for example: `project-4-a` |
+| Comment | *Optional.* A short description of what the project is for. |
```

Comment:
I cannot see the _comment_ field for a project in the latest release.

Reply:
Morning @Artur-Piechuta 
Please see: https://alfresco.atlassian.net/wiki/spaces/PM/pages/951025904/Alfresco+Process+Automation+7.3.0#Release-name-and-release-comment-are-displayed-in-the-Modeling-Application
I have seen the comment field last week on the demo site however it does not appear to be there now. 
Any ideas?

Reply:
I have checked both screens now and added in the correct bits to the Project release page. 
There some minor updates also needed to the Projects page because things had changed and I dont think Docs were informed. 
Let me know what you think, cheers.